evansmusomi/burger-builder
# Handles Sign Up and Sign In via Redux

*What this does*
- Adds auth redux actions and reducer
- Adds sign in screen

*How to verify*
- A user can sign up and sign in to their account
- A developer can verify that auth actions are handled correctly using dev tools
